### PR TITLE
Exchanges: Remove LakeBTC exchange implementation

### PR DESCRIPTION
**Pull request by thrasher-**
# PR Description

This PR removes LakeBTC as it seems their API has been down for quite some time with no notifications on their social media accounts. I've also emailed their support with no response yet.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by shazbert — CHANGES_REQUESTED** [review]
go mod tidy `go-pusher` is unused.

**Review by shazbert — APPROVED** [review]
purge approved! tACK!

**Review by MadCozBadd — APPROVED** [review]
tACK!!! 👋 lakebtc

File: cmd/apichecker/updates.json [hunk 1]
```diff
@@ -275,23 +275,6 @@
...
-     "Val": "flash-message",
-     "TokenDataEnd": "h2",
-     "TextTokenData": "h1",
-     "RegExp": "APIv\\d{1}",
-     "CheckString": "de2491b95ef1f6ea334247b13f0f14f6816fb5961cc63acc0542b07fc0336dd8",
-     "Path": "https://www.lakebtc.com/s/api_v2"
-    }
-   },
-   "Disabled": false
-  },
```
Comment (gloriousCode) [inline cmd/apichecker/updates.json:294]: There is a committed file for the api checker `backup.json` that still mentions LakeBTC if we're going for wide-scale destruction

**Review by gloriousCode — COMMENTED** [review]
⚰️ ☠️ 💸

File: cmd/apichecker/updates.json (hunk 1 above)
Comment (thrasher-) [reply cmd/apichecker/updates.json:294]: nuke.exe fully loaded, similar to crypto markets ATM 👀 😆

**Review by gloriousCode — APPROVED** [review]
tACK! Look at those lovely tests✔️

**Review by shazbert — APPROVED** [review]
changes ACK

**Review by MadCozBadd — APPROVED** [review]
looks good!

**Pull request merged**